cli/parse_pdfs.py

```diff
@@ -73,7 +78,7 @@ def copy_input_to_output_pdf(
                 extra={
                     "props": {
                         "document_id": task.document_id,
-                        "output_path": output_path,
+                        "output_path": str(output_path),
```

Comment:
These str conversions potentially aren't needed now that the cast function isn't used on the path?